components/user-mgt/org.wso2.carbon.user.mgt/src/main/resources/META-INF/services.xml

```diff
@@ -26,92 +26,71 @@
         </parameter>
 
         <operation name="addUser">
-            <parameter name="AuthorizationAction" locked="true">/permission/admin/configure/security/usermgt/users
-            </parameter>
-
+            <parameter name="AuthorizationAction" locked="true">/permission/admin/manage/identity/user-mgt/create</parameter>
         </operation>
 
         <operation name="addUsers">
-            <parameter name="AuthorizationAction" locked="true">/permission/admin/configure/security/usermgt/users
-            </parameter>
-
+            <parameter name="AuthorizationAction" locked="true">/permission/admin/manage/identity/user-mgt/create</parameter>
         </operation>
+
         <operation name="deleteUser">
-            <parameter name="AuthorizationAction" locked="true">/permission/admin/configure/security/usermgt/users
-            </parameter>
-        
+            <parameter name="AuthorizationAction" locked="true">/permission/admin/manage/identity/user-mgt/delete</parameter>
         </operation>
 
         <operation name="addCredential">
-            <parameter name="AuthorizationAction" locked="true">/permission/admin/configure/security/usermgt/passwords
-            </parameter>
-   
+            <parameter name="AuthorizationAction" locked="true">/permission/admin/manage/identity/user-mgt/create</parameter>
```

Comment:
I think this is an update operation wrt a user. So the user should have "/permission/admin/manage/identity/user-mgt/create" permission